### ISSUE TITLE
В бандле лежит не-минифицированный css

### ISSUE THREAD

**Issue by nulladdict** [skbkontur/retail-ui#2881]
Привет!

В `@skbkontur/react-ui` любой версии с css-in-js в бандле лежат не-минифицированные строки с css-in-js, которые занимают лишнее место в бандле.
Вот пример с [tech.skbkontur.ru](https://tech.skbkontur.ru/react-ui/):

<img width="657" alt="исходный код с созданием строк в css-in-js" src="https://user-images.githubusercontent.com/26379644/165295252-10f24f58-c6fa-4573-bb10-f3bc5804aadf.png">

**Comment by zhzz** [comment]
Да, спасибо за issue.  Нужно таки втащить [плагин](https://emotion.sh/docs/@emotion/babel-plugin).

**Comment by nulladdict** [comment]
я правильно понимаю, что плагин на самом деле нужно втащить каждому потребителю, а не в самой библиотеке?

**Comment by zhzz** [comment]
не, думаю, что в библиотеке

**Comment by mshatikhin** [comment]
Добавил минификацию (\n) ушли, но заменого профита не увидел, вес библиотеки стал меньше на 6КБ

**Comment by nulladdict** [comment]
6Kb меньше качать и меньше парсить в рантайме. Может и несущественно но полезно

**Comment by mshatikhin** [comment]
> 6Kb меньше качать и меньше парсить в рантайме. Может и несущественно но полезно

это на всю либу :) а не на рантайм, крч надо еще помучить эту тему
